### PR TITLE
ESP-07 module support

### PR DESCRIPTION
**Pull request by srg74**
ESP-07 module support plus 2 library dependency for Dallas sensor

### CODE REVIEW

File: platformio.ini [hunk 1]
```diff
@@ -43,7 +44,9 @@ lib_deps_external =
   #Timezone@1.2.1
   #For use SSD1306 0.91" OLED display uncomment following
   #U8g2@~2.27.2
-
+  #For Dallas sensor uncomment following 2 lines
+  #DallasTemperature@~3.8.0
+  #OneWire@~2.3.5
```
Comment (srg74) [inline platformio.ini]: Dallas sensor libraries added for temperature sensor as Quinled pub feature

File: platformio.ini [hunk 2]
```diff
@@ -156,6 +159,18 @@ build_flags =
...
+platform = ${common:esp8266.platform}
+monitor_speed = ${common.monitor_speed}
+upload_speed = ${common.upload_speed}
+framework = ${common.framework}
+build_flags =
+  ${common.build_flags}
+  ${common:esp8266.build_flags}
+lib_deps =
+  ${common.lib_deps_external}
+
```
Comment (srg74) [inline platformio.ini]: ESP-07 and ESP-07S module support for external antenna feature

File: platformio.ini [hunk 3]
```diff
@@ -10,6 +10,7 @@ lib_dir = ./wled00/src
 ; env_default = nodemcuv2
 ; env_default = esp01
 ; env_default = esp01_1m
+; env_default = esp07
```
Comment (srg74) [inline platformio.ini]: ESP-07 and ESP-07S module support for external antenna feature

**Comment by Aircoookie** [comment]
Thank you! I do believe that turning off the display is a good idea to save power and prolong lifetime.
However, I don't believe permanently turning off the display 5 minutes after boot is a bit overkill. Displayed values could be needed after that point. I've changed that so that it turns off 3 minutes after any displayed information changed, and back on once it changes again.
I hope you agree :)

**Pull request merged**